konig-dao-sql-runtime/src/main/java/io/konig/sql/runtime/BigQueryFusionChartService.java

```diff
@@ -0,0 +1,79 @@
+package io.konig.sql.runtime;
+
+/*
+ * #%L
+ * Konig DAO SQL Runtime
+ * %%
+ * Copyright (C) 2015 - 2017 Gregory McFall
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.FieldValue;
+import com.google.cloud.bigquery.QueryRequest;
+import com.google.cloud.bigquery.QueryResponse;
+import com.google.cloud.bigquery.QueryResult;
+
+import io.konig.dao.core.ChartGeoLocationMapping;
+import io.konig.dao.core.DaoException;
+
+public class BigQueryFusionChartService {
+	private BigQuery bigQuery;
+	
+	public BigQueryFusionChartService(BigQuery bigQuery){
+		this.bigQuery = bigQuery;	
+	}
+	
+	public ChartGeoLocationMapping getFusionIdMapping() throws DaoException {
```

Comment:
Please add an issue to the Issue Tracker for initializing the util.FusionMapping table.   Our deployment script should create this table and load it with data.